### ISSUE TITLE
Set the git config user.email on clones repositories

### ISSUE THREAD

**Issue by stickycode** [Branchout/branchout#21]
I have many different identities so I cannot set the user.email globally for git it per project. 

Having this configured by branchout would be very handy

**Comment by stickycode** [comment]
I would expect something like this

For the case where this is not set
```
branchout init git@github.com/ExampleOrganisation/example
branchout clone new-project
Please enter your git email: michael@stickycode.net
Please enter your git user: Michael McCallum
Saving settings in ~/branchout/ExampleOrganisation/Branchoutrc
Cloning new
Cloning new-project
Pulled      new                    master
Pulled      new-project        master
```

Looking at the settings
```
cat ~/branchout/ExampleOrganisation/Branchoutrc
```
We find these values now

    GIT_AUTHOR="Michael McCallum"
    GIT_AUTHOR_EMAIL="michael@stickycode.net"

```

**Comment by fredcooke** [comment]
I agree, this would be a great feature, but unless some of your identities are pseudonyms then it's fine to set your name in your global file and set email on particular repos. I have name set globaly and have had for years, and a script on my path for fred.set.git.email and company.set.git.email and clientX.set.git.email etc depending on the machine and work and scenario etc.

**Comment by stickycode** [comment]
I set name globally too.

**Comment by fredcooke** [comment]
We should make this happen. I have some additional ideas about it to share with you, too.
